Makefile

```diff
@@ -37,5 +37,11 @@ docker-sidecar:
 docker-testground:
 	docker build -t iptestground/testground:edge -f Dockerfile.testground .
 
-test:
+clean:
+	rm -rfi $${TESTGROUND_HOME}/plans/placebo
+	rm -rfi $${TESTGROUND_HOME}/plans/example
```

Comment:
I really dislike deleting anything outside the working copy. Let's not do this. The `|| true` below should be enough.

Reply:
-i is for confirming. `placebo` and `example` are really Testground plans.

Reply:
`|| true` is not enough if you have an outdated plan in your TESTGROUND_HOME.

Reply:
You're right that, `-i` is probably not good enough of an excuse to acll `rm` outside of the working dir... maybe TESTGROUND_HOME is not set and a user tries to delete /plans/placebo...

I don't like that our tests depend on TESTGROUND_HOME, maybe it is worth actually fixing this.